during_workshop_11.md

```diff
@@ -306,6 +306,21 @@ You can check whether you have been successful by using the Azure Portal to see
 
 AcmeSub need to be able to translate their subtitles into multiple different languages. One way we could architect this is to have our initial function queue up a function instance per language to do the translation. To help us achieve this we are going to be using Azure Queue storage.
 
+<details><summary>Take a moment to consider what trade-offs this design choice might have</summary>

```

Comment:
Stolen from the alternative workshop, but I think it's good to provide some stronger justification as to why we're building this architecture